build.gradle.kts

```diff
@@ -24,7 +24,7 @@ object Versions {
 
     // Deps
     const val JUNIT_5 = "5.9.3"
-    const val PARTIQL = "1.2.1"
+    const val PARTIQL = "1.2.2-SNAPSHOT"
```

Comment:
self-review: use the version specified by the gradle.properties of partiql-lang-kotlin -- https://github.com/partiql/partiql-lang-kotlin/blob/fe9be5d09f4b18e37b221ee5e8c465cf73053555/gradle.properties#L2